diff -urN so3g-0e351dfc22d3dd195cd5630fb7ac012a3563b4ca_orig/CMakeLists.txt so3g-0e351dfc22d3dd195cd5630fb7ac012a3563b4ca/CMakeLists.txt
--- so3g-0e351dfc22d3dd195cd5630fb7ac012a3563b4ca_orig/CMakeLists.txt	2021-02-25 08:07:09.000000000 -0800
+++ so3g-0e351dfc22d3dd195cd5630fb7ac012a3563b4ca/CMakeLists.txt	2021-07-08 12:43:48.375863733 -0700
@@ -1,5 +1,5 @@
 cmake_minimum_required (VERSION 3.1)
-project (so3g)
+project(so3g LANGUAGES C CXX)
 
 include(local.cmake OPTIONAL)
 
@@ -82,6 +82,7 @@
   # This variable set is needed by check_function_exists; starting in
   # cmake v3.18 you can say BLAS::BLAS instead of the lib path...
   set(CMAKE_REQUIRED_LIBRARIES ${BLAS_LIBRARIES})
+  set(CMAKE_REQUIRED_FLAGS ${OpenMP_CXX_FLAGS})
   check_function_exists(cblas_sgemm CBLAS_OK)
   if(${CBLAS_OK})
     message("-- cblas bindings are included in the BLAS library")
